src/Bicep.Core/LanguageConstants.cs

```diff
@@ -12,6 +12,10 @@ namespace Bicep.Core
 {
     public static class LanguageConstants
     {
+        public const string LanguageId = "bicep";
+        public const string LanguageFileExtension = ".bicep";
+        public const string DevAssemblyFileVersion = "dev";
```

Comment:
Could me move this to Bicep.Core.UnitTests? I wouldn't want anyone to accidentally take a dependency on this in the release code.